README.md

```diff
@@ -21,8 +21,7 @@ To make sure you are running the latest version, you can rebuild the Binder `qp`
 
 You can also browse the (un-executed) notebooks here in this repo:
 
-* **[Basic  Demo](http://htmlpreview.github.io/?https://github.com/aimalz/qp/blob/html/demo.html)** [(raw notebook)](https://github.com/aimalz/qp/blob/master/docs/notebooks/demo.ipynb)
-* **[KL Divergence  Illustration](http://htmlpreview.github.io/?https://github.com/aimalz/qp/blob/html/kld.html)** [(raw notebook)](https://github.com/aimalz/qp/blob/master/docs/notebooks/kld.ipynb)
+* **[Basic  Demo](http://htmlpreview.github.io/?https://github.com/LSSTDESC/qp/blob/html/demo.html)** [(raw notebook)](https://github.com/LSSTDESC/qp/blob/master/docs/notebooks/demo.ipynb)
```

Comment:
I'm getting a 404:Not Found for this link, did your rearrangement of the locations of the notebooks switch this around somehow?

Reply:
Oh, whoops, I saw demo.html in "old_docs" and thought it was the old "pre-fork" demo without all of the updates. I see now that it is actually the updated file, so 
http://htmlpreview.github.io/?https://github.com/LSSTDESC/qp/blob/issue/43/cleanup/old_docs/notebooks/demo.html
is a working link (we'll obviously want to switch the branch we're pointing to to  master for the commit, so
http://htmlpreview.github.io/?https://github.com/LSSTDESC/qp/blob/master/old_docs/notebooks/demo.html
would be the proper link).  However, is `old_docs` the best location for the rendered notebook, or should we maybe move it to the `nb` colocated with the actual ipynb file?

Reply:
Hmm, or if the copy in `old_docs` is not the newest, did you forget to include the new rendered html version of the notebook when you pushed?  I don't see any other copies of demo.html

Reply:
new rendered notebooks copied to docs, we'll deal with rendering in an action in a future PR